### PR TITLE
AAP-566: Revised IDS Guide

### PR DESCRIPTION
**Pull request by lmalivert**
@cbudz Ready for your review.
@mwerner2113 

IDS guide ready to be published.

- Due date for availability in customer portal: Sept 28, 2021

- Link to draft: [Automating Network Intrusion Detection System (IDS) Rules with Ansible](https://docs.google.com/document/d/1VU0BpbM8RUvchmYmjYdZslkgtvM1BJZV6ZT4Cbrklig/edit#heading=h.3424n8r5fwup)

- Link to Jira: https://issues.redhat.com/browse/AAP-566

### CODE REVIEW

**Review by TarikFD — APPROVED** [review]
This build worked for me and the content looks good. PR approved.

-Kevin

File: downstream/modules/security/proc-creating-ids-rule.adoc [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+         name: "ansible_security.ids_rule"
+       vars:
+         ids_rule: 'alert tcp any any -> any any (msg:"Attempted /etc/passwd Attack"; uricontent:"/etc/passwd"; classtype:attempted-user; sid:99000004; priority:1; rev:1;)'
+         ids_rules_file: '/etc/snort/rules/local.rules'
+         ids_rule_state: present
+----
++
+Tasks are components that make changes on the target machine. Since you are using a role that defines these tasks, the `include_role` is the only entry you need.
++
+The `ids_rules_file` variable specifies a defined location for the `local.rules` file, while the `ids_rule_state` variable indicates if the rule should be created if it does not already exist.
```
Comment (ariordan-redhat) [inline downstream/modules/security/proc-creating-ids-rule.adoc:80]:
```suggestion
The `ids_rules_file` variable specifies a defined location for the `local.rules` file, while the `ids_rule_state` variable indicates that the rule should be created if it does not already exist.
```

File: downstream/modules/security/proc-creating-ids-rule.adoc [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+$ ansible-galaxy install ansible_security.ids_rule
+----
+
+. Create a new playbook file titled `add_snort_rule.yml`. Set the following parameters:
++
+----
+- name: Add Snort rule
+  hosts: snort
+----
+
```
Comment (ariordan-redhat) [inline downstream/modules/security/proc-creating-ids-rule.adoc:39]: Would it make sense to group the code before `tasks` into one example code block?

File: downstream/modules/security/proc-creating-ids-rule.adoc [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+----
+$ ansible-navigator run add_snort_rule.ym --mode stdout
+----
++
+Once you run the playbook, all of your tasks will be executed in addition to your newly created rules. Your playbook output will confirm your PLAY, TASK, RUNNING HANDLER, and PLAY RECAP.
+
+
+.Verification
+
+To verify that your IDS rules were successfully created, SSH to the Snort server and look for the content of the ``/etc/snort/rules/local.rules` file.
```
Comment (ariordan-redhat) [inline downstream/modules/security/proc-creating-ids-rule.adoc:93]:
```suggestion
To verify that your IDS rules were successfully created, SSH to the Snort server and view the content of the `/etc/snort/rules/local.rules` file.
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
Just an optional suggestion, and a syntax correction in the final sentence.

**Pull request merged**